dev/CommonStyles/ToggleButton_themeresources.xaml

```diff
@@ -274,6 +274,9 @@
                                         <ObjectAnimationUsingKeyFrames Storyboard.TargetName="ContentPresenter" Storyboard.TargetProperty="BorderBrush">
                                             <DiscreteObjectKeyFrame KeyTime="0" Value="{ThemeResource ToggleButtonBorderBrushChecked}" />
                                         </ObjectAnimationUsingKeyFrames>
+                                        <contract7Present:ObjectAnimationUsingKeyFrames Storyboard.TargetName="ContentPresenter" Storyboard.TargetProperty="BackgroundSizing">
+                                            <DiscreteObjectKeyFrame KeyTime="0" Value="OuterBorderEdge" />
```

Comment:
Why is the OuterBorderEdge value set here? Shouldn't it be set in ContentPresenter?

Reply:
@karenbtlai according to the design, neutral buttons should have **InnerBorderEdge** background sizing while accent buttons (like Toggled ToggleButton) should have **OuterBorderEdge**. Thus, we only change it to outer in the "accent" states :)